applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1020,6 +1020,14 @@ static void stream_configured_cb(struct bt_bap_stream *stream,
 		LOG_INF("%s sink stream configured", headsets[channel_index].ch_name);
 		le_audio_print_codec(headsets[channel_index].sink_stream.codec_cfg,
 				     stream->ep->dir);
+		LOG_DBG("Configured Stream info: %s, %p, dir %d", headsets[channel_index].ch_name,
+			(void *)stream, stream->ep->dir);
+
+		ret = headset_pres_delay_find(headsets[channel_index].sink_ep, &new_pres_dly_us);
```

Comment:
There is a build error as well, you need to set new_pres_dly_us on line:1010 to 0 when initializing it.

Reply:
See above comments, moved the search call back for both source and sink.